.travis.yml

```diff
@@ -22,10 +26,11 @@ before_install:
   - conda update -q conda
 
 install:
-  - conda create -n testenv --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY_VERSION pandas=$PANDAS_VERSION scipy=$SCIPY_VERSION libgfortran=$LIBGFORTRAN_VERSION
+  - conda create -n testenv --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION pandas=$PANDAS_VERSION
```

Comment:
When we run tests on travis/appveyor, do we want to pin transitive dependencies, like numpy?

Reply:
I'd removed them for simplicity, but I suppose for test consistency it makes sense to pin them. Added these back, lmk if these dependency versions should be different